<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1131</width>
    <height>680</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1131</width>
    <height>680</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1131</width>
    <height>680</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Propiedades de Archivos</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1111</width>
      <height>631</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="txtDir">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>511</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="txtProp">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>90</y>
       <width>681</width>
       <height>511</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDire">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>20</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QListWidget" name="lstWidCont">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>331</width>
       <height>511</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblEtiqueta">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Direcctorio:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Contenido</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>70</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Propiedades</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>txtDir</tabstop>
  <tabstop>btnDire</tabstop>
  <tabstop>lstWidCont</tabstop>
  <tabstop>txtProp</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
